db/migrate/20210721200625_create_employee_events.rb

```diff
@@ -0,0 +1,15 @@
+class CreateEmployeeEvents < ActiveRecord::Migration[5.2]
+  def change
+    create_table :employee_events do |t|
```

Comment:
This table is named `employee_events`, but most of the columns (with the exception of `time_off_employees`) appear to be describing an employee, not an event.